packages/components/src/Card/src/Card.module.css

```diff
@@ -0,0 +1,24 @@
+.hop-Card {
+    /* Internal Variables */
+    --border-radius: var(--hop-shape-rounded-md);
+
+    /* Main */
+    --hop-Card-main-border: var(--hop-space-10) solid var(--hop-neutral-border-weak);
```

Comment:
@fraincs do we usually using spacing tokens for border width? we hardcode the 0.0625rem no?

Reply:
Totally missed this, we don't currently use core tokens.

Reply:
I thought we started using the core token since it's actually available now. but we'd normall add it as a variable up top like ---hop-Card-border-size: var(--hop-spacce-10).
It's the only core token we use because we don't have semantic ones for border size.
Even if we don't use the core token, I'd say we should still add the variable up top.